gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileSystem.java

```diff
@@ -56,10 +58,21 @@ public static CloudStorageFileSystem forBucket(String bucket) {
    * @see #forBucket(String)
```

Comment:
Unrelated nit, `@see java.nio.file.FileSystems#getFileSystem(java.net.URI)` could be replaced with `@see java.nio.file.FileSystems#getFileSystem(URI)` as `java.net.URI` is already imported.